helm/volsync/Chart.yaml

```diff
@@ -66,7 +66,7 @@ kubeVersion: "^1.20.0-0"
 # This is the chart version. This version number should be incremented each time
 # you make changes to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: "0.8.0"
```

Comment:
Can you roll back this change? I don't want to modify the default version to v0.9.0 at this point

Reply:
I'm fine doing that, I merely changed it because of that instruction `This version number should be incremented each time you make changes to the chart` following semVer.

I will submit a PR soon.

Reply:
I've removed the chart version bump.
Looks like some formatter fixes were applied automatically, are they OK to keep?

Reply:
Thanks for that - you're right normally we would update the version, but we'll do it at the same time when we make a v0.9 release.  I think the formatter fixes are probably ok.